utils/generate_profile.py

```diff
@@ -0,0 +1,198 @@
+#!/usr/bin/env python3
+
+# Get input file
+# Determine format (CSV or something else like OSCAL)
+# Parse input into a data structure (dictionary)
+
+import abc
+import argparse
+import os
+import re
+
+import json
+import yaml
+from pycompliance import pycompliance
```

Comment:
This is based on the latest content in https://github.com/rhmdnd/pycompliance

If folks are ok with that approach, I'll package it and release to PyPI.

Reply:
I wasn't sure if we'd want to put all the parsing in the pycompliance library. I kept it pretty limited so that it only focused on the common parts of my script and the script from https://github.com/ComplianceAsCode/content/pull/10469